scripts/concurrent_task_utils.py

```diff
@@ -42,6 +44,54 @@ def log(message, show_time=False):
             python_utils.PRINT(message)
 
 
+class TaskResult(python_utils.OBJECT):
+    """Output stream for concurrent_task_utils."""
+
+    def __init__(self, name, failed, error_messages, full_messages):
+        """Constructs a TaskResult object.
+
+        Args:
+            name: str. Name of the check running.
+            failed: bool. Status of the check currently running.
+            error_messages: list(str). List of summary messages returned by
```

Comment:
How would anyone know what is summary messages here? Please make it clear.

Something like:  `List of error messages that are stripped to keep XYZ in the message`.